src/js/components/organisms-page/hero-cover/hero-cover.ts

```diff
@@ -0,0 +1,22 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { HeroCover as IHeroCover } from '@ultimaker/ultimaker.com-model-definitions/dist/organisms/hero/HeroCover';
+import ViewportUtility from 'utils/viewport';
+
+@Component({
+    name: 'hero-cover',
```

Comment:
Naming, HeroCover? Just to conver to model def

Reply:
Lets schedule a meeting ;-)